weblate/checks/chars.py

```diff
@@ -137,6 +139,28 @@ def get_fixup(self, unit):
         return [(' *$', replacement)]
 
 
+class DoubleSpaceCheck(TargetCheck):
+    """Doublespace check."""
+
+    check_id = 'double_space'
+    name = _('Double space')
+    description = _('Translation contains double space')
+    severity = 'warning'
+
+    def check_single(self, source, target, unit):
+        # One letter things are usually decimal/thousand separators
+        if len(source) <= 1 and len(target) <= 1:
+            return False
+        if not source or not target:
+            return False
+
+        # Check if target contains double space
+        return bool(re.search(' {2,}', target))
```

Comment:
```suggestion
        return '  ' in target
```

I think this will perform way better than doing regexp search.